src/services/LoginService.ts

```diff
@@ -0,0 +1,17 @@
+import { LoginParams, VerifyOtpParams } from "types/login"
+
+import { apiService } from "./ApiService"
+
+export const sendLoginOtp = async ({ email }: LoginParams): Promise<void> => {
+  const endpoint = `/auth/login`
+  return apiService.post(endpoint, { email })
+}
+
+export const verifyLoginOtp = async ({
+  email,
+  otp,
+}: VerifyOtpParams): Promise<void> => {
+  const endpoint = `/auth/verify`
+  const res = await apiService.post(endpoint, { email, otp })
+  return res.data
```

Comment:
this returns `any` and doesn't fit with the function signature, where it states that this returns a `Promise<void>` (ie, it **does not** return anything) - could we either change the function signature to match or omit the return here?

Reply:
right, changed in 0ac23591

Reply:
hmm, i think changing it to `Promise<any>` is counter to the intent of ts (type safety in our codebase + preserve knowledge within code thru types) - perhaps we should investigate and put the actual type here?

Reply:
Oh actually on further investigation, the type is actually `Promise<void>` - the endpoint doesn't actually return anything. I've corrected the return type and the functionality in 5d279c98

Reply:
As an aside, think it's clearer w/o the return as it's doing a side effect but this is ok too